### PR TITLE
Implement schema placeholders

### PR DESCRIPTION
**Pull request by tijmenb**
This PR sets us up to be able to add more structured data to all the formats. It:

- Adds placeholders for all formats
- Adds a ArticleSchema supertype
- Eases the current restriction on when we add the schema to the page. We now also add it when there's no publishing organisation or image (which is "required" by Google, [but isn't really](https://stackoverflow.com/questions/41414163/google-requiring-certain-properties-for-certain-types-when-adding-schema-org-mar)).

https://trello.com/c/RLQI1IUV

### CODE REVIEW

File: app/presenters/schema_org/news_article_schema.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
+module SchemaOrg
```
Comment (sihugh) [inline app/presenters/schema_org/news_article_schema.rb]: Github doesn't seem to have recognised this as a move :-/

File: app/presenters/gone_presenter.rb [hunk 2]
```diff
@@ -10,4 +10,9 @@ def initialize(content_item, requested_content_item_path = nil)
   def page_title
     "No longer available"
   end
+
+  # Gone pages do not need a schem because they're not useful to search engines
```
Comment (sihugh) [inline app/presenters/gone_presenter.rb]: `schem` --> `schema`

File: app/presenters/unpublishing_presenter.rb [hunk 3]
```diff
@@ -10,4 +10,9 @@ def initialize(content_item, requested_content_item_path = nil)
   def page_title
     "No longer available"
   end
+
+  # Unpublishing pages do not need a schem because they're not useful to search engines
```
Comment (sihugh) [inline app/presenters/unpublishing_presenter.rb]: `schem` --> `schema`

**Comment by tijmenb** [comment]
Closing this, I'm going for a different approach!

**Pull request closed without merging**